README.md

```diff
@@ -49,4 +47,31 @@ dmx512:
-  * `update_interval`: Specify a custom update interval, i.e. the minimum time between resending the current values (in ms, default 500)
\ No newline at end of file
+  * `update_interval`: Specify a custom update interval, i.e. the minimum time between resending the current values (in ms, default 500)
+
+Outputs point to channels in the DMX universe, from 1 to 512. Some DMX fixtures can be daisy chained with XLR cables, and each one can be set to operate at a different address. When you specify the output channel in the configuration of this component, take into account that _channel (in esphome component) = address + channel (on the fixture)_. When you choose addresses for your DMX fixtures keep in mind that the next free address number on the bus that can be assigned is the _address of the previous light + the last channel number of that light_. 
+
+For easeier understanding, eg. the [ADJ VBar Pak](https://d295jznhem2tn9.cloudfront.net/ItemRelatedFiles/8659/vbar_pak.pdf) can be used in various modes. When you have 4 of this daisy chained, if you set them up to 8-channel mode (to access all the effects) you can set the address of the first one 1, the next one to 9, the third one to 17, fourth one to 25, using the buttons on each fixture. Then in the config you'd have channels for the first one from 1 to 8, the second one from 9 to 16, third one from 17 to 24, forth one from 25 to 32. See [another example](example_4x_adj_vbar_pak.yaml) for this kind of configuration.
+
+The `output` implements float values between 0 and 100. You can use it not only for `light`, but for any component which can send data to it, moreover, it can be set from lambdas. The above example shows use cases for this, where certain outputs are used for hardware effects selection on the DMX fixture, using a `select` component, and parameters can be adjusted from `number` entities.
+
+## Wiring
+
+You can use an RS485-TTL adapter module to connect your ESP devce with the DMX bus.
+
+![MAX485-M](https://user-images.githubusercontent.com/1550668/149642143-7e13fb00-29fd-4e9d-8f11-6b4a2a2bd0ba.png)
+
+For this adapter, use the wiring below:
+
+```
+MAX485-M VCC     -> ESP +5V
+MAX485-M GND     -> ESP GND
+MAX485-M RE & DE -> ESP +3.3V
+MAX485-M DI      -> ESP32 GPIO5 or ESP8266 GPIO2 (as per examples above)
+MAX485-M A       -> XLR 3 (DMX +)
+MAX485-M B       -> XLR 2 (DMX -)
+MAX485-M GND     -> XLR 1 (DMX GND)
+```
+
+Don't forget about 120Ohm termination resistors. If your fixture has DMX IN and OUT ports, on the OUT port or the last fixture in the chain you should use a termination resistor between XLR pins 2 and 3. Similarly on MAX485-M, it has to be placed in parallel with A and B outputs, given that it's going to be placed at the start of the chain.     
+Using good quality 120Ohm impedance cables, DMX lines can be run a maximum distance of approximately 1000 metres. With CAT5 cable DMX lines are safe until approximately 300 meters.
```

Comment:
Are you sure about 1000 metres?

Reply:
Does the MAX really support 3.3V logic when powered by 5V? I would suggest the MAX3485 instead which is directly powered by 3.3V.

Reply:
Empirically tested - works. Powered at either 5 or 3.3V.

Reply:
> Are you sure about 1000 metres?

Yes, with proper cables. I work in the industry.